README.md

```diff
@@ -164,7 +168,59 @@ If you converted your own guest, please use your original credentials.
 ssh -p 10022 root@localhost
 ```
 
-3. Verify TDX is enabled in the guest.
+### Boot TD Guest with virsh(Libvirt)
+
+1. Configure libvirt and guest xml file
+
+It is recommended to run virsh with normal user. In this case, please make sure that the user is added to /etc/libvirt/qemu.conf, for example:
+
+```bash
+# Add current user to /etc/libvirt/qemu.conf
+echo "user = '$USER'
+group = '$USER'
+dynamic_ownership = 0
+security_driver = 'none'
+" | sudo tee -a /etc/libvirt/qemu.conf
+
+# Restart libvirtd service
+systemctl restart libvirtd
+systemctl status libvirtd
+```
+Modify the provided XML file: tdx/guest-tools/td_guest.xml, replace the value of "source file" in td_guest.xml with the absolute path of your guest image.
+
```

Comment:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp; NOTE: It is recommended that you run virsh as a non-root user. To do that, add the user to `/etc/libvirt/qemu.conf`.  Follow these steps:

&nbsp;&nbsp;&nbsp;&nbsp; * Add current user to `/etc/libvirt/qemu.conf`
```bash
echo "user = '$USER'
group = '$USER'
dynamic_ownership = 0
security_driver = 'none'
" | sudo tee -a /etc/libvirt/qemu.conf

&nbsp;&nbsp;&nbsp;&nbsp; * Restart the `libvirtd` service
```bash
systemctl restart libvirtd
systemctl status libvirtd
```

&nbsp;&nbsp;&nbsp;&nbsp; * Edit the provided XML file `tdx/guest-tools/td_guest.xml` and replace the value of `source file` with the absolute path of your guest image.